manifest.php

```diff
@@ -27,7 +27,7 @@
     'label' => 'Development Tools',
     'description' => 'Developer tools that can assist you in creating new extensions, run scripts, destroy your install',
     'license' => 'GPL-2.0',
-    'version' => '6.10.0',
+    'version' => '6.11.0',
     'author' => 'Open Assessment Technologies',
     'requires' => [
         'generis' => '>=11.1.0',
```

Comment:
We need to update tao-core version too :)

Reply:
In requires? mmm then I will do when merge other PR because this can change throw the time

Reply:
Yes, we release tao-core first, then we update here :)